int8 enable
float32 threshold
---
bool success
